pymarketstore/client.py

```diff
@@ -90,30 +111,32 @@ def _request(self, method, **query):
             raise
 
     def query(self, params):
-        if not isiterable(params):
```

Comment:
deleted `isiterable` check because same check is done in `build_query` function after this line